sdks/python/apache_beam/examples/inference/large_language_modeling/pipeline_utils/utils.py

```diff
@@ -0,0 +1,98 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements. See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership. The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License
+import apache_beam as beam
+import torch
+from apache_beam.io.filesystems import FileSystems
+from apache_beam.ml.inference.pytorch_inference import PytorchModelHandlerTensor
+from transformers import AutoTokenizer
+from transformers import T5ForConditionalGeneration
+
+
+# ModelHandlerwrapper can be removed once:
+# https://github.com/apache/beam/issues/22572 is fixed
+# [START PytorchModelHandlerTensorWrapper]
+class ModelHandlerWrapper(PytorchModelHandlerTensor):
+  """Wrapper to PyTorchModelHandlerTensor to load the
+    weights in model.model instead of model."""
+  def load_model(self):
+    model = self._model_class(**self._model_params)
+    model.to(self._device)
+    file = FileSystems.open(self._state_dict_path, 'rb')
+    model.model.load_state_dict(torch.load(file))
+    model.eval()
+    return model
+
+
+# [END PytorchModelHandlerTensorWrapper]
+
+
+# Modelwrapper can be removed once:
+# https://github.com/apache/beam/issues/22572 is fixed
```

Comment:
Same as above comment. 

We can pass `generate` to inference_fn param in the ModelHandler.

Something like this https://github.com/apache/beam/blob/09606de8779ca284666581c7c2da314df4f9ddb1/sdks/python/apache_beam/ml/inference/pytorch_inference_test.py